docs/reference/commandline/create.md

```diff
@@ -55,7 +55,12 @@ Creates a new container.
       --memory-swap=""              Total memory (memory + swap), '-1' to disable swap
       --memory-swappiness=""        Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.
       --name=""                     Assign a name to the container
-      --net="default"               Set the Network mode for the container
+      --net="bridge"                Connects a container to a network
+                                    'bridge': creates a new network stack for the container on the docker bridge
+                                    'none': no networking for this container
+                                    'container:<name|id>': reuses another container network stack
+                                    'host': uses the host network stack inside the container
+                                    'NETWORK': connects the container to user-defined network using `docker network create` command, either the name or id can be used.
```

Comment:
looking at this again, perhaps we should use a similar approach as `container:`;

```Markdown
'<network-name|network-id>': connects the container to a user-defined network.
```

wdyt?